applications/advanced-concepts/tags-and-placeholders.md

```diff
@@ -6,17 +6,19 @@ description: Tags and placeholders in machine translation
 
 **Tags** and **placeholders** are pieces of non-translatable code inside the input text.
 
-
 ## Tags
 
 Tags format, structure and annotate texts, and can also be used as placeholders. Tags are usually XML, for example HTML.
 
 There are two approaches for working with tagged inputs:
 
-Tags can be kept in the training data and at run time. In this approach, the machine translation can misplace, corrupt or drop tags in the output.
-Tags can be removed from the training data and at run time, and then reinserted after translation. Tag reinsertion is challenging because it is inefficient and impacts on metrics.
+1. Tags are kept in the training data and at runtime. This can affect the quality of the (non-tagged) machine translation output.
+2. Tags are removed in the training data and at runtime, and reinserted after translation.
+
+With either approach, tags may be misplaced, corrupted or dropped in the output.
+
 
-### Example
+#### Example
```

Comment:
Examples may be making this article too long, but I think they make the topic much clearer. What do you think?

Reply:
It's good.

To make it clearer:

- Use some kind of formatting.  Make Example in a blockquote (inset), not a section.  And make the actual segment text in Italics.

- Say "Spanish" somewhere.

- Use an example that is more about tags.  The *<service> cuenta* example can also be interpreted as a literal translation (not about the tag being misplaced).  It's more like a placeholder.

I'd expect something like this:

> ### Example
> > English original:
> > ```A <b>red</b> car```
> >
> > Expected Spanish translation:
> > ```Un carro <b>rojo</b>```
> >
> > Bad translation:
> > ```Un <b>rojo</b> carro```
> >
> > Good translation, but misplaced tag:
> > ```Un <b>carro</b> rojo```
> >
> > Good translation, but corrupted tag:
> > ```Un carro b rojo b```